net.peercoin.peercoin-qt.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
has this been submitted upstream?

Reply:
Yes, https://github.com/peercoin/peercoin/blob/master/contrib/debian/peercoin-qt.desktop

Reply:
so it should be better to get upstream directly

Reply:
It seems I can't go with upstream .desktop file because of the file name referenced in it.

> Icon=net.peercoin.peercoin-qt

vs

> Icon=/usr/share/pixmaps/peercoin128.png

Reply:
you can:
1. patch it (and submit the patch) - since it's invalid
2. use the desktop file tools to modify it at post-install.

There is also "rename-icon" but I'm not sure how it will work in this specific case

Reply:
Done.